### PR TITLE
v0.9.0 release

### PR DESCRIPTION
**Pull request by jakepruitt**
Just to get some :eyes: on this before I fast-forward merged it onto master. Let me know if anything needs updating, or if we want to make the jump to v0.9.0.

@tmcw @mourner @scothis

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 An in-progress version being developed in the `master` branch.
 
+## 0.8.2 (Jul 24 2015)
+
+* `glyphs` URL now normalizes without the `/v4/` prefix for `mapbox://` urls. Legacy behavior for `mapbox://fontstacks` is still maintained (#1385)
+* expose `geojson-vt` for GeoJSON sources (#1271)
```
Comment (mourner) [inline CHANGELOG.md]: Expose geojson-vt _options_

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+* bearing snaps to "North" within a tolerance of 7 degrees (#1059)
+* Now you can directly mutate the minzoom and maxzoom layer properties with map.setLayerZoomRange(layerId, minzoom, maxzoom)
+* Control is now exposed
+* More examples
+* Refactored handlers
```
Comment (mourner) [inline CHANGELOG.md]: They're not just refactored, but also configurable individually with Map options https://github.com/mapbox/mapbox-gl-js/blob/master/js/ui/map.js#L136-L142

Comment (mourner) [inline CHANGELOG.md]: Also with the ability to enable/disable them individually at runtime, e.g. `map.scrollZoom.disable()`

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
 
 An in-progress version being developed in the `master` branch.
 
+## 0.8.2 (Jul 24 2015)
+
+* `glyphs` URL now normalizes without the `/v4/` prefix for `mapbox://` urls. Legacy behavior for `mapbox://fontstacks` is still maintained (#1385)
+* expose `geojson-vt` for GeoJSON sources (#1271)
+* bearing snaps to "North" within a tolerance of 7 degrees (#1059)
+* Now you can directly mutate the minzoom and maxzoom layer properties with map.setLayerZoomRange(layerId, minzoom, maxzoom)
+* Control is now exposed
```
Comment (mourner) [inline CHANGELOG.md]: lets add more context — "exposed `mapboxgl.Control`, a base class used by all UI controls".

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,17 @@
 
 An in-progress version being developed in the `master` branch.
 
+## 0.8.2 (Jul 24 2015)
+
+* `glyphs` URL now normalizes without the `/v4/` prefix for `mapbox://` urls. Legacy behavior for `mapbox://fontstacks` is still maintained (#1385)
+* expose `geojson-vt` for GeoJSON sources (#1271)
+* bearing snaps to "North" within a tolerance of 7 degrees (#1059)
+* Now you can directly mutate the minzoom and maxzoom layer properties with map.setLayerZoomRange(layerId, minzoom, maxzoom)
```
Comment (mourner) [inline CHANGELOG.md]: backticks around code

**Comment by mourner** [comment]
Also no mention of batch-related changes by @scothis in the changelog.

**Comment by mourner** [comment]
The changelog also misses some other improvements like https://github.com/mapbox/mapbox-gl-js/commit/88d17565cb18409bd910e622716b47ea3961b876, https://github.com/mapbox/mapbox-gl-js/pull/1283, https://github.com/mapbox/mapbox-gl-js/commit/8dc3fe5d33ad69b588ba0aa580275e179d31554e, https://github.com/mapbox/mapbox-gl-js/commit/2a2aec44a39e11e73bdf663258bd6d52b83775f5, https://github.com/mapbox/mapbox-gl-js/commit/8f2561dc131dad48ebdf52d24759b10bd129aa19, https://github.com/mapbox/mapbox-gl-js/commit/001c7b9a91bfa58e798cfd5bb26b1904aec8d969, https://github.com/mapbox/mapbox-gl-js/commit/d5cd6c689c35d312cd468a9de0960689e86e6d85

File: README.md [hunk 5]
```diff
@@ -7,8 +7,8 @@ A WebGL JavaScript interactive maps library that can render [Mapbox Vector Tiles
 Include the source via HTML tags:
 
 ```html
-<script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.8.1/mapbox-gl.js'></script>
-<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.8.1/mapbox-gl.css' rel='stylesheet' />
+<script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.9.0/mapbox-gl.js'></script>
+<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.9.0/mapbox-gl.css' rel='stylesheet' />
```
Comment (lucaswoj) [inline README.md]: Should this be 9.0 or 8.2?

Comment (lucaswoj) [inline README.md]: The PR is titled "v0.8.2 release" but the version seems to have been changed to "0.9.0". I think the former is more desirable.

Comment (mourner) [inline README.md]: I think there are enough new features to justify a minor version bump. Why do you think 0.8.2 is better?

Comment (lucaswoj) [inline README.md]: @mourner No strong opinion. v0.9.0 is good.

**Pull request merged**